dali/pipeline/executor/executor.h

```diff
@@ -338,18 +325,17 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunMixed() {
   DeviceGuard g(device_id_);
 
   auto mixed_idx = QueuePolicy::AcquireIdxs(OpType::MIXED);
```

Comment:
```suggestion
  auto mixed_idxs = QueuePolicy::AcquireIdxs(OpType::MIXED);
```
Sorry for nitpicking, but when it's `auto`-typed, we need a name that makes cardinality clear.

Reply:
Done